docs/migration_cleanup.md

```diff
@@ -119,12 +119,12 @@ The `migration` config_split disables the tracking of content changes.
 
 First rebuild the snapshot table.
 ```bash
-drush --uri=$DRUSH_OPTIONS_URI drush cs:s
+drush --user=1 --uri=$DRUSH_OPTIONS_URI cs:s
```

Comment:
```suggestion
drush --uri=$DRUSH_OPTIONS_URI cs:s --user=1
```

Since `--user` isn't global needs to be on the end of the command but correct in that it is missing here.